cleanlab_studio/studio/studio.py

```diff
@@ -396,6 +397,93 @@ def download_embeddings(
         """
         return np.asarray(api.download_array(self._api_key, cleanset_id, "embeddings"))
 
+    def create_enrichment_project(
+        self,
+        name: str,
+        dataset_id: str,
+    ) -> enrichment.EnrichmentProject:
+        """
+        Creates a Cleanlab Studio Enrichment Project.
+
+        Args:
+            name (str): Name of the enrichment project to create.
+            dataset_id (str): ID of dataset to be enriched.
+
+        Returns:
+            EnrichmentProject: The [EnrichmentProject](../enrichment#class-enrichmentproject) object
+            for the new enrichment project.
+        """
+        enrichment_project_dict = api.create_enrichment_project(
+            api_key=self._api_key,
+            name=name,
+            dataset_id=dataset_id,
+        )
+
+        return enrichment.EnrichmentProject(
+            api_key=self._api_key,
+            id=enrichment_project_dict["id"],
+            name=enrichment_project_dict["name"],
+            created_at=enrichment_project_dict["created_at"],
+        )
```

Comment:
+ `updated_at`. i know the update time is not necessary. but we should keep all four fields across all methods. i.e., id, name, created_at, updated_at.

Reply:
I decided to not store `updated_at` so that whenever a user attempts to look up this field they get the correct value.